docs/management_api.yml

```diff
@@ -110,6 +110,33 @@ paths:
           description: Internal server error.
           schema:
             $ref: "#/definitions/Error"
+  /devices/{id}/auth/{aid}:
+    delete:
+      summary: Remove the device authentication set
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          format: Bearer [token]
+          description: Contains the JWT token issued by the User Administration and Authentication Service.
+        - name: id
+          in: path
+          description: Device identifier.
+          required: true
+          type: string
+        - name: aid
+          in: path
+          description: Authentication data set identifier.
+          required: true
+          type: string
+      responses:
+        204:
```

Comment:
if it's management, are we considering 401 here? or is it 'internal management', meaning it will only be called via deviceadm, so auth will lready be verified?

Reply:
It's management. We are not including 401 in responses of any of the endpoints.
I think that we have information in the docs that every management endpoint can return 401.

Reply:
oh ok.